advanced_source/static_quantization_tutorial.py

```diff
@@ -158,35 +150,35 @@ def __init__(self, num_classes=1000, width_mult=1.0, inverted_residual_setting=N
                 [6, 320, 1, 1],
             ]
 
-        # only check the first element, assuming user knows t,c,n,s are required
+        # 사용자가 t, c, n, s가 요구된다는 것을 안다는 전제하에 첫 번째 요소만 확인하기
         if len(inverted_residual_setting) == 0 or len(inverted_residual_setting[0]) != 4:
             raise ValueError("inverted_residual_setting should be non-empty "
                              "or a 4-element list, got {}".format(inverted_residual_setting))
 
-        # building first layer
+        # 첫 번째 계층 만들기
         input_channel = _make_divisible(input_channel * width_mult, round_nearest)
         self.last_channel = _make_divisible(last_channel * max(1.0, width_mult), round_nearest)
         features = [ConvBNReLU(3, input_channel, stride=2)]
-        # building inverted residual blocks
+        #  반전된 나머지 블락 만들기
         for t, c, n, s in inverted_residual_setting:
             output_channel = _make_divisible(c * width_mult, round_nearest)
             for i in range(n):
                 stride = s if i == 0 else 1
                 features.append(block(input_channel, output_channel, stride, expand_ratio=t))
                 input_channel = output_channel
-        # building last several layers
+        # 나머지 여러 계층들 만들기
```

Comment:
```suggestion
        # 나머지 계층 만들기
```

국문에서는 복수 형태가 조금 부자연스러운 것 같습니다.

Reply:
원문의 정확한 의미 파악을 위해 복수형으로 번역하는 게 나을 것같습니다.